auth_signup_verify_email/controllers/main.py

```diff
@@ -34,7 +34,7 @@ def passwordless_signup(self, values):
             values["email"] = values.get("login")
 
         # Remove password
-        values["password"] = False
+        values["password"] = ""
```

Comment:
Hmm I don't say it doesn't work, but I suspect it works also with previous implementation, and it's not a good practice to store empty strings in fields. I don't feel comfortable skipping good practices precisely in the password field 🤔, so please elaborate this need before merging.